examples/python/workflow.py

```diff
@@ -22,6 +22,7 @@
     intensity = builder.create_intensity(recipe)
```

Comment:
I would delete this file and just work with the notebooks

Reply:
Yes, but would require to write an additional notebook, about fitting/optimizing. That part of this script is not yet documented in the form of a notebook, only generating data.

Reply:
Moved to separate https://github.com/ComPWA/tensorwaves/issues/68, as I prefer to limit this PR to the addition of generating notebook output on the RTD pages (with a notebook about generating data as a (necessary) example).

Reply:
Ok good